sdk/src/Services/DynamoDBv2/Properties/AssemblyInfo.cs

```diff
@@ -18,6 +18,7 @@
 #else
 #error Unknown platform constant - unable to set correct AssemblyDescription
 #endif
+[assembly: InternalsVisibleTo("AWSSDK.UnitTests.DynamoDBv2.NetFramework, PublicKey=0024000004800000940000000602000000240000525341310004000001000100db5f59f098d27276c7833875a6263a3cc74ab17ba9a9df0b52aedbe7252745db7274d5271fd79c1f08f668ecfa8eaab5626fa76adc811d3c8fc55859b0d09d3bc0a84eecd0ba891f2b8a2fc55141cdcc37c2053d53491e650a479967c3622762977900eddbf1252ed08a2413f00a28f3a0752a81203f03ccb7f684db373518b4")]
```

Comment:
FYI this file is generated, it'll be overwritten by the generator the next time it runs.

I do see why you had to add it since a lot of the `Search` related types are internal, but we may need a different approach here (e.g. would it be better to add an integration test instead? Or revisit the access modifiers to make them public?)

Reply:
That could work, but I think you'd also need to update the test projects to be signed... I'm honestly not a fan of making internals visible; I know we did that for `Core` but I wish we didn't (which is why I mentioned maybe using an integration test instead if possible).

We can until Norm's back to get his opinion.

Reply:
Discussed with @dscpinheiro and I'm okay with relying on `InternalsVisibleTo` to make it easier to write unit tests. The generator will need to be updated to write this `InternalsVisibleTo` since the `AssemblyInfo.cs` is generated.

When doing the generator update you will need to add the logic to check if it is generating the DynamoDB `AssemblyInfo` and if so add the attribute. I don't think we want to add this attribute to every service's `AssemblyInfo`.